src/react/5-styling-in-react.md

```diff
@@ -48,7 +48,7 @@ One of the downsides of this approach is the performance cost associated with ma
 
 Like normal HTML, JSX elements can accept a class attribute (called `className`). Classes can either be located in a global style sheet linked in the index.html file, or be directly imported into a component (see below).
 
-This import syntax can be used in any modern web-loader.
+This import syntax can be used in any modern web-loader. React uses `webpack` to accomplish this.
```

Comment:
React is not coupled to webpack in any way. There's a few alternatives. I think you meant to say "Create React App uses webpack".

Reply:
Done